src/transformLocalImports.ts

```diff
@@ -96,6 +96,10 @@ function resolveModule(
         return `${module}.js`;
     }
 
+    if (module.startsWith('swiper/react') && !module.endsWith('.js')) {
```

Comment:
What version of swiper do you use?

Reply:
For the current version of swiper these changes will be incorrect. Use `import {...} from 'swiper/resct.js';` in your code instead.

Reply:
I think this pulll request can be closed.